cluster/manifests/02-visibility/01-namespace.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: visibility
+  labels:
+    admission.zalando.org/infrastructure-component: "true"
```

Comment:
This is what protects the `visibility` namespace. We could do the same for `kube-system` but we don't have a YAML for that at the moment.